test/private-key.test.js

```diff
@@ -10,6 +10,7 @@ describe('private-key', function () {
 
   beforeEach(function () {
     privateKey = 'I AM PRIVET KEY!?!!~1!'
+    privateKey64 = Buffer.from(privateKey).toString('base64')
```

Comment:
If this is only used in the one test, could it be moved to the describe block below?